notes-manager/src/js/reducers/index.js

```diff
@@ -0,0 +1,37 @@
+import { ADD_NOTE } from '../constants/action-types';
+import { REMOVE_NOTE } from '../constants/action-types';
+
+const initalState = {
+    notes: [],
+};
+
+function findIndex(notes, note_to_remove) {
```

Comment:
Look into the `_.without` method of `underscore.js`

https://lodash.com/docs/4.17.11#without